### ISSUE TITLE
What is the definition of a shot?

### ISSUE THREAD

**Issue by fluppeteer** [etwa-rsc/rules#2]
This was triggered by a question about this pot:

https://youtu.be/09TSwVSl3ZM?t=339

In this, the wink leaves the squidger, bounces off the pot, bounces off the squidger, and is potted. I imagine this is legal, rather than "interference with a wink in flight", and certainly it has to be legal to touch winks that are already moving because they're in the pile being played. On the other hand, bouncing out of the pot and hitting a squidger probably *is* interference.

I (Andrew) have long had a complaint that "quick and continuous" is an insufficient indication of a legal shot. My preference would be that "the motion of the squidger must be brief enough that the player has no time to react to the movement of winks during the shot" - or similar. "Quick" of itself is meaningless, and "continuous" is impossible when the movement of a squidger contacts multiple winks successively, as in a gromp. In this case, so long as the player has not had the chance to react to the movement of the wink, multiple impacts with the squidger are part of the same (legal) shot.

A related argument is that some players are willing to "rest on" the wink at the end of the shot, whereas a strict reading of the rules (and practice of some other players) is to lift off a click shot; this effectively turns every click shot into an air shot, if you don't have time to react to the movement of the upper wink in ensuring that you're not resting on the lower one.

Can we clean this up?

**Comment by timhunt** [comment]
The current wording of this rule is:

> For a shot to be a legal shot, the following criteria must be met.
10.1 The squidger must first touch the upper surface of a free wink of the colour to be played. For the purposes of this rule, the upper surface of a wink is that part of the wink that is visible from directly above.
> 10.1.1 The colour to be played is that whose turn it is, except for the special case of a free shot with a nominated colour described in Rule 22.6.1.
> 10.1.2 The player can choose which one of the free winks of the appropriate colour to play. For instance, the person controlling blue may choose to play separate blue winks from behind the baseline on successive blue turns. Alternatively the person may choose to play the same blue wink on successive blue turns even if there are unplayed blue winks behind the baseline.
> 10.2 From the moment when a wink starts to move irreversibly, the movement of the squidger must be quick and continuous until contact between squidger and winks ceases.
> 10.3 If the first wink touched by the squidger is part of a pile, during the shot the squidger must not touch any other winks in the same pile that were not vertically below some part of the first wink played at the start of the shot.
> 10.4 During the act of causing wink or winks to move, the player must not touch the wink played with hand, body or clothing, nor may the player touch with hand, body or clothing any other winks that are part of the same pile as the first wink contacted by squidger.
> 10.5 At the start of the downward motion of squidger immediately before contact with a wink is made, the lowest point of the player’s squidger must not be more than 5 cm above the highest point of the first wink to be touched.
> 10.6 Winks behind baselines must be played one at a time, and from a position where they rest only on the mat.

And it is extremely difficult to word any better than that. The current wording is (I believe) essentially what Charles Relle came up with many years ago. For example the 2001 rules (the oldest I have) contain most of the same key phrases:

> (b) When playing a shot, a player must hold the squidger not more than 5 cm above the highest point of the first wink squidged. A shot consists of downward pressure of squidger on wink that is an attempt to move a wink or that causes a wink to move irreversibly. For the purposes of this rule, a movement is irreversible if, when the squidger ceases contact with the wink, all winks do not return to the position they occupied before contact with the squidger began. The squidger must first touch the upper surface (that part of the wink that is visible from directly above) of an unsquopped wink (see Rule 6) of the correct colour in sequence. If the wink is squopping all or part of a pile, the squidger may subsequently touch only winks vertically below some part of the wink first played. It must not touch other winks in the same pile. From the moment when a wink starts to move irreversibly, the movement of the squidger must be quick and continuous. A shot may consist of tapping a wink so that another moves from beneath it, but it is a foul shot to squeeze a wink from beneath the top wink and then play the top wink if the movements are distinct. The shot ends when contact between the squidger and playable winks ceases, and all winks have come to rest. Any shot that does not comply with these criteria is a foul shot (see Rule 12).

**Comment by fluppeteer** [comment]
My objection is, and has always been, that "quick and continuous" is gibberish.

What is "quick"? Less than a second? Less than a minute? So instantaneous that many Bristols would be illegal?

What is "continuous"? Can you clatter down the back of a gromp, or is it two shots the moment the squidger slides off the upper wink? Does the trajectory of the squidger have to be linear, which it effectively isn't for many phone card shots and pile flips? If you play a click shot and hit the bottom wink, is that illegal? If you play a click shot and stick on the bottom wink accidentally, should you have time to lift off, or is it illegal inherently?

We've spent twenty years looking at shots and compiling group knowledge on what's considered okay - and I've still had Rupert Wilson rant at me about perfectly legal shots and scare off OUTS novices in the process. I've always disliked that anyone not learning winks in the context of an established club has no idea how to interpret these rules. Now we have clubs starting up in remote locations and people trying to get new players to take up the game on their own, we should be clearer.

**Comment by hatj2** [comment]
> A shot consists of a player exerting downward pressure of squidger on wink.
9.1 In order for a shot to be played, either the squidger must cause irreversible movement of at least one wink or the player must be attempting to cause irreversible movement of at least one wink.
9.1.1 For the purposes of this rule, a movement is irreversible if, when the squidger ceases contact with the wink, any wink does not return to the position it occupied before contact with the squidger began.
9.1.2 Touching a wink with a squidger is not a shot if the wink does not move irreversibly and there was no intention for the wink to move.
9.1.3 If winks are moved without the player exerting downward pressure of squidger on wink, then this does not constitute a shot but is deemed to be interference with winks (Rule 24). For instance, this applies when a person drops a squidger onto a wink or moves a wink with his or her finger while pointing to it.
9.2 A shot ends when contact between squidger and winks ceases and all winks have come to rest.

This doesn't *explicitly* say that you need to keep hold of the squidger throughout the shot - we should probably insert that somewhere.

**Comment by hatj2** [comment]
> 9.1.2 Touching a wink with a squidger is not a shot if the wink does not move irreversibly and there was no intention for the wink to move.

I think this should read "... if no wink moves irreversibly... … for any wink to move."

**Comment by fluppeteer** [comment]
Agreed to the "no wink"/"any wink" thing. I'm not sure there's anything saying you have to hold the squidger at all - or indeed how, since there may be an argument for holding a squidger with tweezers for a delicate shot. I'm a little wary of mandating something in case we discriminate against people with hand deformities who actually need mechanical assistance.

I do think clarifying the distinction between touching moving winks as part of the shot (as in the pot in the video) and interfering with a moving wink (e.g. catching a wink that's rolling off by putting your squidger in front of it) should be made somehow; perhaps contact between wink and squidger should occur only in positions attained by the squidger during its "quick and continuous" (separate issue) motion, but the travel of winks does not have to meet the same definition of "quick". Although you could still have a wink land on a squidger in unusual circumstances.

**Comment by fluppeteer** [comment]
Patrick argues that the "wink bouncing off the pot, then the squidger, then being potted" thing in the video is illegal, and that he's seen Andy pot nurdled winks the way he claimed to have done it. I am concerned that it is highly likely in pile play that some winks will have moved due to pressure from higher winks and still get hit by the squidger - I could believe this of many gromps. So I don't think "you can't touch a moving wink at all" is workable.

Patrick doesn't like the idea of adjudicating based on someone's reaction time, and preferred to say that "quick and continuous" means "you can't observe two separate movements of winks". I'm not convinced. Partly, I think it just punts the confusion from "quick" to "separate"; partly I don't think it's true in a long split dock (the squdger may be moving a long way fast with no reasonable time for the player to react, but you can still see winks moving distinctly); partly I think it restores an ambiguity about whether you're allowed to rest on a lower wink that's not moving (irreversibly).

There was a suggestion that we should disallow contact with the underside of winks at all, which might discount some highly curved squidger motions from "hooking" winks, especially with a concave squidger. As someone who plays weird pile shots (because I get squopped up a lot and need to attempt hero shots to sort out the mess), I think this could be an active change, and I'm not entirely sure I'm in favour.

**Comment by timhunt** [comment]
By they way, I think that is an illegal pot too, and I can pot that sort of wink legally (as can Andy, normally, which is why the infraction was not caught at the time.)

**Comment by fluppeteer** [comment]
Ok. If we don't want to allow that kind of shot (I don't hugely mind), saying that the squidger and winks cannot be in contact after the "quick" period of squidger motion works for me. I do think "can't touch a wink that's already moving" is unworkable, though.

I do wonder about the "blow a pile, end up stuck on the lower wink" situation. Is it always illegal, because contact with winks continued? Is it legal because there was no motion of the wink? If we rely on my "reactions" guideline, do you always have to play hard through to the mat just in case you stick?

I imagine we might have a lot of "that was illegal because we stuck, but we can't be bothered to reconstruct the pile so we'll accept it" situations. That may not be a bad thing.

**Comment by timhunt** [comment]
Getting stuck on the bottom wink after a pile break is in the same category of resting on the bottom wink after a click. It is illegal. If you are in any doubt about the legality of the shot, you should lift you squidger up at the end.

The kind of obviously illegality we want to rule out are:

* The illegal pot in the video.
* A shot where you phone-card the wink high into the air, then use your squidger to bat it away.

To be honest, the traditional "quick and continuous" handles most of this. Anything after that is covered by the traditional "interfering with the winks in motion".

I guess the exact ambiguity to nail down is at what points the winks become 'in motion' and the squidger is no longer able to touch them. Certainly, after bouncing off the pot, I think.

**Comment by hatj2** [comment]
Of course, there's a difference between shots that are difficult to play legally (eg pile shots where there are things in the way of the squidger) and shots that are quite easy to play legally but sometimes aren't (eg the pot in the video, the pile break where you stick on a wink at the bottom); the latter will be ignored because we think the player would play it legally with ease on the second attempt, but the former is the one where we need shot judges and so on. It might also be ignored if the pile is hard to reconstruct. 
It's also worth asking what happens when the two sides disagree on whether the shot was legal - do the opponents get the benefit of the doubt and get the replay?

**Comment by timhunt** [comment]
We should not have different guidelines about what to do if the players can't agree etc. in different places. There should be a single statement, and what we currently have is Rule 26, particularly 26.1.

> 26. Etiquette Rules
> 
> 26.1 If the players are unable to agree on any matter concerned with the play of the game, or are in doubt as to the meaning or interpretation of any rule, they must if possible call a competent person to act as an umpire. If any player is doubtful whether a proposed shot will be played legally, an umpire must be called before the shot is played. The umpire may advise on the likely legality of the shot. If the shot is played, the umpire must decide if it has been legally executed. The decision of the umpire is final on all matters.

**Comment by fluppeteer** [comment]
(Sorry for the silence - been off sick.)

I'm sure we can construct a borderline case when it comes to hitting a wink bouncing off a pot. Some of it comes under "be reasonable", but I think it's expected that newcomers to the game (or Rupert W) may need some guidance on what "reasonable" looks like - hence my "didn't have time to make anything else happen" interpretation re. reaction time. I do accept that a ricochet off the pot onto a stationery squidger isn't alleviated by the reaction time thing, though. Just because it's a can of worms doesn't mean I wouldn't like to see it opened.

Re. umpires, I feel obliged to bring up the Varsity match wherein an umpire was called to adjudicate whether a wink (let's call it red) was squopping another (let's call it yellow). The umpire decreed that yellow was squopping red, and retroactively deemed by both teams not to have met the definition of "competent person". I imagine we ought to elaborate slightly - perhaps "a person acceptable to both teams to act in the role". _I've_ umpired, and I'm certainly not competent; a competent person may still be unacceptable (as in the period when Matt was known for consistently decreeing all borderline cases squopped).
